src/Shaders/ShadersInclude/defaultFragmentDeclaration.fx

```diff
@@ -0,0 +1,80 @@
+uniform vec4 vDiffuseColor;

```

Comment:
just a reminder: i changed the build mechanism a bit so you will need to reference these includes for both pbrmaterial and stdmaterial in their config section :)